docs/resources/runbook.md

```diff
@@ -32,35 +32,36 @@ resource "firehydrant_runbook" "example-runbook" {
   description = "This is an example runbook"
   owner_id    = firehydrant_team.example-owner-team.id
   attachment_rule = jsonencode({
-    "logic" = {
-      "eq" = [
```

Comment:
oh interesting the key doesn't need to be quoted, strange was this something that autoformatted for you, as mine wasn't removing it.

Reply:
nah, i just happened to see it when i was in my editor running fmt after i added automatic to some examples. 
i think either way is fine (as long as your key doesn't start with a number, then it has to be quoted), we just have to pick one way or the other. 
since i wrote all of the json in the huge doc PR without quotes, i just removed quotes from here because it was faster. we could def go the other way. do you have any thoughts on which on would be easier to understand?

Reply:
The number part is a bit ugly with the user_data bag beginning with a number. However it doesn't have to because that value could be anything as long as it matches in the logic with `user.[value]`. Definitely highlighting wise it's easier to see the difference between key and value without the quotes. I went ahead and changed over the `rule` PR to match.